pulsar-client/src/main/java/org/apache/pulsar/client/impl/transaction/TransactionCoordinatorClientImpl.java

```diff
@@ -97,14 +98,9 @@ public CompletableFuture<Void> startAsync() {
                             handler.start();
                         }
                     } else {
-                        handlers = new TransactionMetaStoreHandler[1];
-                        CompletableFuture<Void> connectFuture = new CompletableFuture<>();
-                        connectFutureList.add(connectFuture);
-                        TransactionMetaStoreHandler handler = new TransactionMetaStoreHandler(0, pulsarClient,
-                                getTCAssignTopicName(-1), connectFuture);
-                        handlers[0] = handler;
-                        handlerMap.put(0, handler);
-                        handler.start();
+                        return FutureUtil.failedFuture(new TransactionCoordinatorClientException("The broker "
+                                + "doesn't enable transaction or transaction "
+                                + "has not completed initialization in broker"));
```

Comment:
The broker doesn't enable the transaction coordinator, or the transaction coordinator has not initialized